### PR TITLE
feat(ci): 135 checkov output pre commit

### LINKED ISSUE #135: Improve checkov output in pre-commit.
### Is your feature request related to a problem?

Right now we don't specify which test we want to check, and the result of that is whole errors which checkov makes are skipped because they touch something we don't need or wont to check.

### Describe the solution you'd like

We need to clarify which test we want to do with check and properly configure it.

### PR DESCRIPTION
**Pull request by horiagunica**
## Description

- Modified checkov hook in pre-commit, ignored examples checkov warnings and fixed tflint warnings .
- This PR closes #135 

## Motivation and Context
- More details about specific checkov checks manipulation in #135 .

## How Has This Been Tested?
- Installed necessary plugins and ran "pre-commit run -a" in the base repo folder to run all checks manually.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes if appropriate.
- [x] All new and existing tests passed.

### CODE REVIEW

File: modules/panorama/variables.tf [hunk 1]
```diff
@@ -149,3 +149,20 @@ variable "metadata" {
...
+  description = "IAM Service Account for running Panorama instance (just the email)"
+  default     = null
+  type        = string
+}
+
+variable "scopes" {
+  default = [
+    "https://www.googleapis.com/auth/compute.readonly",
+    "https://www.googleapis.com/auth/cloud.useraccounts.readonly",
+    "https://www.googleapis.com/auth/devstorage.read_only",
```
Comment (lstadnik) [inline modules/panorama/variables.tf:163]: Do the panorama need access to object storage ? or other scopes as well ?

Comment (horiagunica) [reply modules/panorama/variables.tf]: Removed panorama access to storage from the default scopes

File: modules/panorama/variables.tf [hunk 2]
```diff
@@ -149,3 +149,14 @@ variable "metadata" {
   type        = map(string)
   default     = {}
 }
+
+variable "service_account" {
```
Comment (pimielowski) [inline modules/panorama/variables.tf:153]: We stick to the pattern: 
```
variable "example" {
   description = ""
   type        = ""
   default     = ""
 }
```
:)

**Review by pimielowski — APPROVED** [review]
LGTM! 👍

File: modules/panorama/variables.tf (hunk 2 above)
Comment (horiagunica) [reply modules/panorama/variables.tf:153]: Re-arranged the order and also added description !

**Review by migara — APPROVED** [review]

**Pull request merged**